### DISCUSSION TITLE
跪求把米家摄像头的视频流接入ha

### DISCUSSION THREAD

**Discussion by backtimebug** [XiaoMi/ha_xiaomi_home#292] [General]
如题，米家的大部分摄像头都支持私有协议，无法通过rtsp或者onvif等协议接入，希望官方能支持下

**Comment by uilee01** [comment]
官方套件就别想了，不过有些摄像头可以刷机实现本地视频流

**Reply by uilee01** [reply]
这毕竟也涉及安全问题，米家的都得通过云端服务器

**Comment by huhuime** [comment]
树大招风不太可能了

**Comment by SusanPhevos** [comment]
[xiaomi_home 不会支持查看米家摄像机视频流功能](https://github.com/XiaoMi/ha_xiaomi_home/wiki/Features-that-will-not-be-implemented#camera-video-stream)。
